packages/slice-machine/src/features/onboarding/helpers/useOnboardingStepsContent.ts

```diff
@@ -0,0 +1,21 @@
+import { getOnboardingStepsContent } from "@/features/onboarding/content";
+import type {
+  OnboardingStepContent,
+  OnboardingStepContentDefinition,
+  OnboardingStepType,
+} from "@/features/onboarding/types";
+import { useRepositoryInformation } from "@/hooks/useRepositoryInformation";
+
+export function useOnboardingStepsContent(): OnboardingStepContent;
+export function useOnboardingStepsContent(
+  stepId: OnboardingStepType,
+): OnboardingStepContentDefinition;
+export function useOnboardingStepsContent(
+  stepId?: OnboardingStepType,
+): OnboardingStepContent | OnboardingStepContentDefinition {
+  const { repositoryUrl } = useRepositoryInformation();
+  const content = getOnboardingStepsContent({ repositoryUrl });
+
+  if (stepId) return content[stepId];
+  return content;
+}
```

Comment:
So now that we've refactored this, it's clear that this is no longer a hook, but a component: receives some props, renders something.

Reply:
Yeah sound better, refactored!